app/views/components/docs/label.yml

```diff
@@ -3,7 +3,7 @@ description: Use labels for all form fields.
 body: |
   For use with other form inputs e.g. [Radio buttons](/component-guide/radio)
 
-  Forked from the upcoming GOV.UK Frontend, when GOV.UK Frontend release we can replace these source files.
+  Forked from the upcoming [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend), when GOV.UK Frontend release we can replace these source files.
```

Comment:
I would squash this line edit into commit 543cc481fac6339fcaf22b11460df5247c132bf8 where the line was added.